.github/workflows/lint.yml

```diff
@@ -0,0 +1,32 @@
+name: Lint
+
+on: [pull_request_target, push]
```

Comment:
if this what you are referring to with removing push?

Reply:
Correct. `pull_request_target` is a newer feature from them so that it gets triggered on forks and the main branch, but only for PRs. If I keep `push` it gets noisy -- any PRs on the main branch (unforked) will trigger double testing. However, it's one of those things that has to be in `master` to work for `pull_request_target`, so `push` is simulating what it will look like after we merge.